### PR TITLE
Add Nielsen to users page

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

**Comment by mosabua** [comment]
Can you please verify size and potentially optimize the images in this PR @kylefraser .. and send the updated files to @martint so he can add them,.

**Comment by martint** [comment]
I already optimized them myself.

File: _data/users.yml [hunk 1]
```diff
@@ -109,6 +109,9 @@
   logo: /assets/images/logos/netflix.png
   logosmall: /assets/images/logos/netflix-small.png
   #   https://netflixtechblog.com/using-presto-in-our-big-data-platform-on-aws-938035909fd4
+- name: Nielsen
+  logo: /assets/images/logos/nielsen.png
+  logosmall: /assets/images/logos/nielsen-small.png
```
Comment (findepi) [inline _data/users.yml:114]: and no `url`?

**Review by findepi — APPROVED** [review]

**Comment by kylefraser** [comment]
fwiw.. checked and looks good to me 👍

File: _data/users.yml (hunk 1 above)
Comment (martint) [reply _data/users.yml:114]: No, that's only for the ones that have a testimonial.

**Pull request merged**